anime-relations.txt

```diff
@@ -115,6 +121,9 @@
 # Charlotte -> ~: Tsuyoi Monotachi
 - 28999|10103:14 -> 31553|11365:1
 
+# Chi's Sweet Home -> ~ OVA
+- 3841|3293:105 -> 	7972|5166:1
```

Comment:
You used a tab rather than a space, which makes the rule invalid. Did you test it before sending the pull request?

Also, isn't the OVA a sequel to "Chi's Sweet Home: Atarashii Ouchi" rather than "Chi's Sweet Home"?